extension/rusheye/README.adoc

```diff
@@ -0,0 +1,182 @@
+= Arquillian Graphene RushEye Extension
+
+*rusheye* is an extension to Arquillian Graphene platform which provides the possibility to add the visual validation in your page objects/tests. In order to use it, please place this artifact configuration into Maven dependencies:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.jboss.arquillian.graphene</groupId>
+    <artifactId>arquillian-graphene-rusheye</artifactId>
+</dependency>
```

Comment:
this dependency doesn't bring you the necessary implementation - it is just the aggregator. The required dependency is `arquillian-graphene-rusheye-impl`